docs/fastdds/dds_layer/publisher/dataWriterListener/dataWriterListener.rst

```diff
@@ -1,43 +1,46 @@
+.. include:: ../../../../03-exports/aliases.include
+.. include:: ../../../../03-exports/aliases-api.include
+
 .. _dds_layer_publisher_dataWriterListener:
 
 DataWriterListener
 ==================
 
-:class:`DataWriterListener` is an abstract class defining the callbacks that will be triggered
+|DataWriterListener-api| is an abstract class defining the callbacks that will be triggered
 in response to state changes on the :ref:`dds_layer_publisher_dataWriter`.
 By default, all these callbacks are empty and do nothing.
 The user should implement a specialization of this class overriding the callbacks
 that are needed on the application.
 Callbacks that are not overridden will maintain their empty implementation.
 
-:class:`DataWriterListener` defines the following callbacks:
+|DataWriterListener-api| defines the following callbacks:
 
-* **on_publication_matched**: The :ref:`dds_layer_publisher_dataWriter` has found a
+* **on_publication_matched**: The DataWriter has found a
```

Comment:
In general: On the other entities, the callback functions are linked to the API. I think it is a good idea and the same should be done here.